src/components/RssPreviewCard/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useRouter } from "next/router"
+
+import type { RSSItem } from "@/lib/types"
+
+import { cn } from "@/lib/utils/cn"
+import { isValidDate } from "@/lib/utils/date"
+
+import { BaseLink } from "../ui/Link"
+
+type RssPreviewCardProps = RSSItem & {
+  className?: string
+}
+
+const RssPreviewCard = ({
```

Comment:
There are some similarities with `EventPreviewCard`. Wondering if we could move some of the shared styles to the base card component. Also wondering if all the cards now will have this hover effect (if not then I would leave it as is ofc).